Formula/mallet.rb

```diff
@@ -0,0 +1,29 @@
+class Mallet < Formula
+  desc "MAchine Learning for LanguagE Toolkit"
+  homepage "http://mallet.cs.umass.edu/"
+  url "http://mallet.cs.umass.edu/dist/mallet-2.0.8.tar.gz"
+  sha256 "5b2d6fb9bcf600b1836b09881821a6781dd45a7d3032e61d7500d027a5b34faf"
+  head "https://github.com/mimno/Mallet.git"
```

Comment:
Please remove the `head` and the `bottle` block here.

Reply:
will do, thanks

Reply:
In fact, this one does not require bottling as neither anything is actually being compiled nor any additional resources are being downloaded. So, `bottle :unneeded`.

Reply:
To be clear, I should add that line, yes? Sorry, quite new to writing formulae.

Reply:
I went ahead and did so.